index.html

```diff
@@ -49,6 +50,182 @@ <h2>LET’S CONNECT</h2>
+                  <li class="tags3"><h3>Javascript</h3></li>
+                </ul>
+              </div>
+              <div class="see-more-detail">
+                <div class="button">
+                  <a href="#">
+                    See Project
+                  </a>
+                </div>
+              </div>
+          </div>
+        </div>
+      </div>
+      <div class="card2">
+        <img src="assets/images/mobile/portfolio2.svg" alt="image">
+        <div class="card-text2">
+          <div class="card-header2">
+            <h3>Multi-Post
+              Stories</h3>
+            <div class="frame2">
+              <ul>
+                <li class="client"><h4>CANOPY</h4></li>
+                <li class="role"><i class="fa fa-circle"></i><h4>Back End Dev</h4></li>
+                <li class="year"><i class="fa fa-circle"></i><h4>2015</h4></li>
+              </ul>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <div class="tags">
+                <ul>
+                  <li class="tags1"><h3>HTML</h3></li>
+                  <li class="tags2"><h3>CSS</h3></li>
+                  <li class="tags3"><h3>Javascript</h3></li>
+                </ul>
+              </div>
+              <div class="see-more-detail">
+                <div class="button">
+                  <a href="#">
+                    See Project
+                  </a>
+                </div>
+              </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    
+    <section class="about">
+      <div class="about-content">
+        <div class="about-content-top">
+          <h3>About <br/> Myself</h3>
+          <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          <div class="social-about">
+            <div class="social-text-about">
+              <h2>LET'S CONNECT</h2>
+            </div>
+            <div class="social-icon-about">
+              <ul>
+                <li><a href="#"><img src="assets/images/mobile/twitter.svg" class="twitter-icon-about" alt="image"></a></li>
+                <li><a href="#"><img src="assets/images/mobile/github.svg" class="github-icon-about" alt="image"></a></li>
+                <li><a href="#"><img src="assets/images/mobile/Linkedin-icon.svg" class="linkedin-icon-about" alt="image"></a></li>
+                <li><a href="#"><img src="assets/images/mobile/m.svg" class="about-m-icon" alt="image-about"></a></li>
+                <li><a href="#"><img src="assets/images/mobile/finger.svg" class="finger-icon-about" alt="image"></a></li>
+              </ul>
+            </div>
+          </div>
+          <div class="resume">
+            <a href="#">
+              Get my resume
+            </a>
+          </div>
```

Comment:
-  [ ] Add more right padding to your 'Get my resume' link to look similar to what is on the template.